### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by gnespolino**
Changes to document TEAs updates for analytics

### CODE REVIEW

File: releases/2.41/README.md [hunk 1]
```diff
@@ -9,6 +9,60 @@
...
+
+Specifically, the script will update the type of these attributes to "TEXT." For instance, if a TEA is declared as a NUMBER, but its value is a text string, the script will modify the attribute type to TEXT.
+
+This enhancement ensures data integrity and alignment between attribute types and their actual values which is expecially needed in Analytics.
+
+##### Verifying which attributes will be affected:
+
+The following query can be executed before upgrading to verify which TEAs will be affected.
+
+The query will also show the update statement (it's not needed to run the update, the system will do automatically on next system startup).
```
Comment (Philip-Larsen-Donnelly) [inline releases/2.41/README.md:26]:
```suggestion
The query will also show the update statement (it's not needed to run the update, the system will do automatically on next system startup).

> Note:
> You can also use this script to identify the TEAs that you need to correct if you *do not* want the type to be automatically changed.
```

**Review by Philip-Larsen-Donnelly — APPROVED** [review]
Very nice detailed information @gnespolino !

I added a small note just to push the message that they can fix things before the upgrade if they don't want it to change their types.

File: releases/2.41/README.md [hunk 2]
```diff
@@ -9,6 +9,63 @@
...
+
+> Note:
+> You can also use this script to identify the TEAs that you need to correct if you *do not* want the type to be automatically changed.
```
Comment (teleivo) [inline releases/2.41/README.md:29]: Using the word `script` for the Flyway migration and the SQL query you post below might be a bit confusing. I would use a different word for each of the two and always use that. Personally I would use `migration` for the Flyway migration as that is the word we generally use AFAIK.

File: releases/2.41/README.md [hunk 3]
```diff
@@ -9,6 +9,63 @@
...
+            from trackedentityattribute tea
+                     join trackedentityattributevalue teav
+                          on tea.trackedentityattributeid = teav.trackedentityattributeid) as t1
+      where safe_to_cast = false
+group by uid, valuetype, description;
+
+DROP function if exists can_be_casted(s text, type text);
+```
+
+##### Usage:
```
Comment (teleivo) [inline releases/2.41/README.md:65]: Usage to me in a readme indicates to me how a user is supposed to use the software. I would not use such a header as users don't have to do anything.

You could use something like https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts at the start saying that a Flyway migration doing X will run automatically when starting DHSI2 in 2.41

File: releases/2.41/README.md (hunk 2 above)
Comment (gnespolino) [reply releases/2.41/README.md]: done

File: releases/2.41/README.md (hunk 3 above)
Comment (gnespolino) [reply releases/2.41/README.md]: nice tip, thank you

**Pull request merged**